prow/prstatus/prstatus.go

```diff
@@ -25,13 +25,12 @@ import (
 	"strings"
 	"time"
 
-	gogithub "github.com/google/go-github/github"
 	"github.com/gorilla/sessions"
 	githubql "github.com/shurcooL/githubv4"
 	"github.com/sirupsen/logrus"
 	"golang.org/x/oauth2"
+	gogithub "k8s.io/test-infra/prow/github"
```

Comment:
nit: `test-infra` imports should be grouped together
non-nit: we should not call this `gogithub`